provider/terraform/tests/resource_compute_subnetwork_iam_test.go.erb

```diff
@@ -245,3 +247,6 @@ resource "google_compute_subnetwork_iam_policy" "foo" {
 }
 `, account, subnetworkName, subnetworkName, region, roleId)
 }
+<% else %>
+// Magic Modules doesn't let us remove files - blank out beta-only common-compile files for now.
```

Comment:
Did you also want to remove iam_compute_subnetwork.go?

Reply:
Right! Forgot those files existed as well as the stuff in `provider.go`.